src/views/ifsBrowser.js

```diff
@@ -687,6 +705,37 @@ module.exports = class IFSBrowser {
 
     return storage.setSourceList(existingDirs);
   }
+
+  /**
+   *
+   * @param {string} searchTerm
+   * @param {string} searchPath
+   */
+  async doSearchInStreamfiles(searchTerm, searchPath) {
+    try {
+      await vscode.window.withProgress({
+        location: vscode.ProgressLocation.Notification,
+        title: t(`ifsBrowser.doSearchInStreamfiles.title`),
+      }, async progress => {
+        progress.report({
+          message: t(`ifsBrowser.doSearchInStreamfiles.progressMessage`, searchTerm, searchPath)
+        });
+
+        let results = await Search.searchIFS(getInstance(), searchPath, searchTerm);
+
+        if (results.length > 0) {
+          results = results.map(a => ({ ...a, label: path.posix.relative(searchPath, a.path) }));
+          setSearchResults(searchTerm, results.sort((a, b) => a.path.localeCompare(b.path)));
+
+        } else {
+          vscode.window.showInformationMessage(t(`ifsBrowser.doSearchInStreamfiles.noResults`, searchTerm, searchPath));
+        }
```

Comment:
As `searchIFS` always returns an array, you can safely perform the `map` and `sort` operations here all at once and make `result` a `const`.
```suggestion
        const results = (await Search.searchIFS(getInstance(), searchPath, searchTerm))
          .map(a => ({ ...a, label: path.posix.relative(searchPath, a.path) }))
          .sort((a, b) => a.path.localeCompare(b.path));

        if (results.length > 0) {
          results = results;
          setSearchResults(searchTerm, results);
        } else {
          vscode.window.showInformationMessage(t(`ifsBrowser.doSearchInStreamfiles.noResults`, searchTerm, searchPath));
        }
```